assets/components/directDebit/helpers/ajax.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { routes } from 'helpers/routes';
+import type { Csrf as CsrfState } from 'helpers/csrf/csrfReducer';
+
+const checkAccount = (
+  sortCode: string,
+  accountNumber: string,
+  accountHolderName: string,
```

Comment:
checkAccount (which proxies bank_details_lookups) specifically shouldn't need the accountHolderName: https://developer.gocardless.com/api-reference/#bank-details-lookups-perform-a-bank-details-lookup

Reply:
Yes that's true, I added `accountHolderName` to this request to match `directDebitData` case class. Additionally, I added this field in `directDebitData` because I will reuse the same case class during the creation of the subscription in Zuora (not implemented in this PR) (https://github.com/guardian/support-frontend/blob/103f68ebd78539241340ee36016e106c9edab8df/app/services/stepfunctions/RegularContributionsClient.scala#L36). 

In order to avoid sending unnecessary data in this request, I see two option:
* Create a specific case class for this request
* Convert accountHolderName to be an Option and do not send it in the CheckAccount request.

I prefer the creation of a new case class just for this request. What are your thoughts on this @paulbrown1982 ? cc: @davidfurey